fe/fe-core/src/main/java/com/starrocks/connector/parser/trino/Trino2SRFunctionCallTransformer.java

```diff
@@ -81,234 +80,253 @@ private static void registerAllFunctionTransformer() {
         registerUnicodeFunctionTransformer();
         registerMapFunctionTransformer();
         registerBinaryFunctionTransformer();
+        registerHLLFunctionTransformer();
         // todo: support more function transform
     }
 
     private static void registerAggregateFunctionTransformer() {
         // 1.approx_distinct
         registerFunctionTransformer("approx_distinct", 1,
-                "approx_count_distinct", ImmutableList.of(Expr.class));
```

Comment:
Why change this? `ImmutableList` is fine if use lower java's version(List.of should be used in higher jdk9)?

Reply:
<img width="885" alt="image" src="https://github.com/StarRocks/starrocks/assets/2594794/fb9313f5-1410-4d67-aeae-c5676dc5d1f5">

Because this problem, it make the code be 'B'. So I change the code. Teacher Li, May the problem be not be resolved?